Description: Fix FTBFS on arm64
 Use correct defines for selecting 32-bit neon assembler code in 
 embedded libpng, so that it is not used on 64-bit armv8.
Author: Wookey <wookey@debian.org>
Last-Update: 2014-09-30

---
 libs/libpng/libpng-src/pngpriv.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- texlive-bin.orig/libs/libpng/libpng-src/pngpriv.h
+++ texlive-bin/libs/libpng/libpng-src/pngpriv.h
@@ -126,7 +126,7 @@
     * or put -DPNG_ARM_NEON_OPT=0 in CPPFLAGS.
     */
 #  if (defined(__ARM_NEON__) || defined(__ARM_NEON)) && \
-   defined(PNG_ALIGNED_MEMORY_SUPPORTED)
+   defined(PNG_ALIGNED_MEMORY_SUPPORTED) && defined(__arm__)
 #     define PNG_ARM_NEON_OPT 2
 #  else
 #     define PNG_ARM_NEON_OPT 0
